content/docs/Configuration/examples.md

```diff
@@ -229,6 +229,27 @@ jobs:
   ```
 {{< /details >}}
 
+{{< details title="Tag cloud resources in Testing Farm" >}}
```

Comment:
I would consider admonition, if Hugo supports that and if it's a requirement

Reply:
It is not required at all

Reply:
> It is not required at all

fine with me then :)

---

had a look around, seems like Hugo doesn't support it by itself anyways :/

Reply:
Ok, Resolving :)